### PR TITLE
fu-406: V3 for Dropdown Component

### PR DESCRIPTION
**Pull request by AndyKIron**

### CODE REVIEW

**Review by danielbrazg — CHANGES_REQUESTED** [review]

File: projects/fusion-ui/components/tags-input/v3/tags-input.component.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import {Component, forwardRef} from '@angular/core';
+import {NG_VALUE_ACCESSOR} from '@angular/forms';
+import {FilterByFieldPipe} from '@ironsource/fusion-ui/pipes/collection';
+import {ClonePipe} from '@ironsource/fusion-ui/pipes/clone';
+import {DropdownService} from '@ironsource/fusion-ui/components/dropdown/servise';
+import {TagsInputBaseComponent} from '@ironsource/fusion-ui/components/tags-input/common/base';
+
+@Component({
+    selector: 'fusion-tags-input',
```
[the remark this answers is not in the document]
Comment (AndyKIron) [reply projects/fusion-ui/components/tags-input/v3/tags-input.component.ts:9]: so, default export will be v2

File: projects/fusion-ui/components/dropdown-dual-multi-select/dropdown-dual-multi-select.component.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ import {
 import {InputSize} from '@ironsource/fusion-ui/components/input/common/base';
 import {ControlValueAccessor, FormControl, NG_VALUE_ACCESSOR} from '@angular/forms';
 import {DynamicComponentConfiguration} from '@ironsource/fusion-ui/components/dynamic-components';
-import {DropdownOption} from '@ironsource/fusion-ui/components/dropdown';
+import {DropdownOption} from '@ironsource/fusion-ui/components/dropdown-option/entities';
```
[the remark this answers is not in the document]
Comment (AndyKIron) [reply projects/fusion-ui/components/dropdown-dual-multi-select/dropdown-dual-multi-select.component.ts:17]: Re-Structure for this Amit do.

File: projects/fusion-ui/components/tags-input/v3/tags-input.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielbrazg) [reply projects/fusion-ui/components/tags-input/v3/tags-input.component.ts]: no. we are not doing default other then v3

[the remark this answers is not in the document]
Comment (AndyKIron) [reply projects/fusion-ui/components/tags-input/v3/tags-input.component.ts]: Oh...

**Pull request merged**